keyboards/4pplet/waffling80/readme.md

```diff
@@ -0,0 +1,18 @@
+# waffling80
+
+A TKL PCB attempting a87 compatibility with different switch and layout-options. 
+
+* Keyboard Maintainer: [4pplet](https://github.com/4pplet)
+* Hardware Supported: [waffling80](https://github.com/4pplet/waffling80)
+
+Make example for this keyboard (after setting up your build environment):
+
+    make 4pplet/waffling60/rev_b:default
```

Comment:
```suggestion
    make 4pplet/waffling80:default
```

Reply:
I updated with "make 4pplet/waffling80/rev_a:default", thanks for the catch! :)